early-attack-detector-py/src/agent.py

```diff
@@ -80,6 +86,21 @@ def initialize():
     BETA = 'beta' in package['name']
     logger.info(f"Beta: {BETA}")
 
+    global TP_ATTACKER_LIST
+    TP_ATTACKER_LIST = get_tp_attacker_list()
+
+    alert_config = {
+        "alertConfig": {
+            "subscriptions": [
+                {
+                    "botId": FUNDING_BOTS[6],
```

Comment:
funding bots has 6 elements, so this will fail
+ if this is for the tp_attacker_list, maybe it shouldn't be as funding bots? since it technically isn't. we can add a different constant

Reply:
disregard the length

Reply:
It's for the hops bot

Reply:
why does it subscribe to the hops but not to the others?

Reply:
@salort-forta , It only subscribes to the "MALICIOUS-FUNDING" alert of the hops bot which is related to the True Positive list (The normal alert from the hops bot is "SUSPICIOUS-FUNDING")

It needs to do so, because so far, the AD factors in funding sources only _after_ a contract has been deemed malicious. Now we need to bypass that logic, so `handle_alert` is used to receive new alerts (related to TP list hops) quickly and have an up-to-date list of attackers